environments/rinkeby/readme.md

```diff
@@ -11,11 +11,53 @@
   - `open.aragonpm.eth`: `0x915c4a47e7c7f7ab04ac70b4bcfba257a1e8b040`
 - DAOFactory (Aragon 0.6): `0x2298d27a9b847c681d2b2c2828ab9d79013f5f1d`
 - DAOFactory (Aragon 0.7): `0xfdef49fbfe37704af55636bdd4b6bc8cd19143f6`
+- DAOFactory (Aragon 0.8): `0xad4d106b43b480faa3ef7f98464ffc27fc1faa96`
 
 ## Deployments
 
 -----------
 
+- Version: Aragon client 0.8
+- Start date: September 3rd 2019, 18.30pm CEST
+- Finish date: TBD
+- Deployer: ([@facuspagnuolo](https://github.com/facuspagnuolo)) [`0x03acbcb547d03c8e7746ef5988012b59604aa083`](https://rinkeby.etherscan.io/address/0x03acbcb547d03c8e7746ef5988012b59604aa083)
+
```

Comment:
Can we add the aOS commit hash here as we usually do for these?

https://github.com/aragon/deployments/blob/a9fc9ba0068f249aea6d686f224762d29edf5ccf/environments/staging/readme.md#aragonos-1